src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day calendar--month-length">
```

Comment:
The class here should be, 

calendar--start-day-wed

You see, with a  **-wed**  at the end

so that the calendar knows where to start.